renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "config:base"
+  ]
```

Comment:
Jeg forslår:
```
{
  "extends": [
    "config:base"
	"schedule:earlyMondays", 
	":pinOnlyDevDependencies"
  ]
}
```

Da får vi PR på mandag, bare dev-deps pinned.

Fordelen med pinned dev-deps er at "works on my machine" problemet blir mindre fordi alle bruker eksakt samme versjon av standard/eslint/rollup. 

Prod-deps bør være semver "~", og ikke konfigurert spesielt for renovate.